spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Decidim
+  module Comments
+    describe CommentsController, type: :controller do
+      routes { Decidim::Comments::Engine.routes }
+
+      let(:organization) { create(:organization) }
+      let(:metadata) { { ceco: "ceco", ceco_txt: "ceco_txt" } }
+      let!(:authorization) { create(:authorization, user: current_user, name: "dummy_authorization_handler", metadata: metadata) }
+      let!(:space_with_other_permissions) do
+        create(
+          :participatory_process,
+          :published,
+          organization: organization,
+          suara_permissions: { ceco: "a", ceco_txt: "b" }
+        )
+      end
+
+      let!(:space_with_permissions) do
+        create(
+          :participatory_process,
+          :published,
+          organization: organization,
+          suara_permissions: { ceco: "ceco", ceco_txt: "ceco_txt" }
+        )
+      end
+
+      let!(:space_without_permissions) do
+        create(
+          :participatory_process,
+          :published,
+          organization: organization,
+          suara_permissions: { ceco: "", ceco_txt: "", tipologia: "" }
+        )
+      end
```

Comment:
Maybe you can test it too when space has one permission but not another, for example: 

```
let!(:space_with_one_permission) do
        create(
          :consultation,
          :published,
          organization: organization,
          suara_permissions: { ceco: "ceco", ceco_txt: "b" }
        )
      end
```